src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -927,8 +926,6 @@ def forward(
         ...     token_type_ids=token_type_ids,
         ...     labels=labels,
         ... )
-
-        >>> loss = outputs.loss
```

Comment:
Same here, please don't remove this, but rather add an expected value instead

Reply:
I'll revert the change. I'd prefer not to add an expected value in this PR, since it's a bit outside the scope of the change, but happy to help in a separate change.